apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/lru/LruCacheStore.java

```diff
@@ -0,0 +1,87 @@
+package com.apollographql.android.cache.normalized.lru;
+
+import android.app.ActivityManager;
+import android.content.Context;
+
+import com.apollographql.android.api.graphql.internal.Optional;
+import com.apollographql.android.cache.normalized.CacheStore;
+import com.apollographql.android.cache.normalized.Record;
+import com.nytimes.android.external.cache.Cache;
+import com.nytimes.android.external.cache.CacheBuilder;
+import com.nytimes.android.external.cache.Weigher;
+
+import java.util.Collections;
+import java.util.Set;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import static android.content.Context.ACTIVITY_SERVICE;
+import static android.content.pm.ApplicationInfo.FLAG_LARGE_HEAP;
+
+public final class LruCacheStore extends CacheStore {
+
+  private final Cache<String, Record> lruCache;
+
+  public static LruCacheStore create(Context context) {
```

Comment:
any reason not to have a time expiration configuration as well?  So for us we want articles to expire every day so that someone doesn't see old news. :-)